build-env/target/.common/debian_packages.conf.sh

```diff
@@ -0,0 +1,120 @@
+# build dependencies
```

Comment:
Why is the directory `.common` hidden?

Reply:
There are two reasons for that:
1. environment search logic skips hidden folder, so in there can be any file and `.common` will not be listed as build environment
2. listing `target` directory  with `ls` command gives list of directories with environments (debian, ubuntu, centos, rhel). If someone would see this directory as regular directory could take this as environment which could be misleading

Reply:
Alternative idea: `COMMON`(all caps) instead of `.common`(hidden)

Reply:
renamed it to `common`. It maybe easier to navigate when it is not hidden. We just have to keep in mind that we can not add `packages.conf.sh` file in it (as this file is used to detect if directory contains environment setup).

Reply:
Cool.  Maybe note that in `common/README`?

Reply:
I just added this information to `README.devel.md` as this is relevant for scripts developers only.